docs/aws/README.md

```diff
@@ -3,31 +3,32 @@
 This guide will provide a complete Karpenter installation for AWS.
 These steps are opinionated and may need to be adapted for your use case.
 
+> This guide should take less than 1 hour to complete and should cost less than $.25
+
 ## Environment
 ```bash
-CLOUD_PROVIDER=aws
+export CLUSTER_NAME=$USER-karpenter-demo
+export AWS_DEFAULT_REGION=us-west-2
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-CLUSTER_NAME=$USER-karpenter-demo
-AWS_DEFAULT_REGION=us-west-2
+KARPENTER_VERSION=$(curl -fsSL \
+  https://api.github.com/repos/awslabs/karpenter/releases/latest \
+  | jq -r '.tag_name')
 ```
 
 ### Create a Cluster
 
-Create an EKS cluster
+Karpenter can run anywhere, including on self-managed node groups, [managed node groups](https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html), or [AWS Fargate](https://aws.amazon.com/fargate/).
```

Comment:
We need another line here, "including other cloud providers"

Reply:
Should that be in the AWS guide? I would assume when we get support for other providers we'll have different guides for those environments.

Reply:
I think @rothgar has the right idea here. I think this will become effectively a non-issue when #484 lands